cves/kernel/CVE-2019-15666.yml

```diff
@@ -167,8 +167,11 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    A series of unit tests could have easily been developed to test the edge
```

Comment:
I don't think unit tests apply here; they're talking mostly about outside tools like a fuzzer, unit tests would have to be aware of the projects domain which in the instructions it says things like that should be false